### ISSUE TITLE
Indicator: urban compactness

### ISSUE THREAD

**Issue by gamalielpalomo** [CityScienceLab-GDL/cityscope-ccu#5]
- [ ] Population density
- [ ] Housing density
- [ ] Daytime activities
- [ ] Night activities
- [ ] Knowledge-dense activities

**Comment by LAAP** [comment]
I agree, Please, let me know what if the issue if any, or how can I help.

I am following the development of the indicators at:
https://docs.google.com/spreadsheets/d/1jKwGme6Be_x_L1tqHRgWMWzzBPTaU0lI/edit#gid=1112802940
